packages/std/src/math/fraction.rs

```diff
@@ -12,3 +12,57 @@ pub trait Fraction<T>: Sized {
     /// If `p` is zero, None is returned.
     fn inv(&self) -> Option<Self>;
 }
+
+impl<T: Clone> Fraction<T> for (T, T) {
```

Comment:
All uXXXX and UintYYYY implement derive Copy. We chould only implement `impl<T: Copy> Fraction<T> for (T, T)` to make the implementation a bit slimmer. But I don't have any strong favourite here. They both work.